com.unity.multiplayer.mlapi/Runtime/Logging/NetworkLog.cs

```diff
@@ -67,6 +67,7 @@ private static void LogServer(string message, LogType logType)
                     writer.WriteStringPacked(message);
 
                     NetworkManager.Singleton.MessageSender.Send(NetworkManager.Singleton.ServerClientId, NetworkConstants.SERVER_LOG, NetworkChannel.Internal, buffer);
+                    NetworkManager.Singleton.NetworkMetrics.TrackServerLogSent(NetworkManager.Singleton.ServerClientId, (uint)logType, (ulong)buffer.Length);
```

Comment:
I'm surprised to see `NetworkManager.Singleton` in here - @TwoTenPvP is this class pending singleton removal?

Reply:
Looks like I missed that class in Singleton Removal..

Reply:
As this is a static call to the NetworkManager, would you rather I add a reference to it into the NetworkLog class or wait for some further changes to make those function calls static?

Reply:
We should probably do this in a separate PR.